server/admin/admin_server_test.go

```diff
@@ -119,42 +124,55 @@ func TestAdminServer_GetTree(t *testing.T) {
 	invalidTree.TreeState = trillian.TreeState_HARD_DELETED
 
 	tests := []struct {
-		desc              string
-		getErr, commitErr bool
+		desc string
+		// getErr is the error that will be returned by the mock AdminTX.GetTree() method.
+		getErr error
+		opts   testOptions
 	}{
 		{
 			desc: "success",
+			opts: testOptions{
+				shouldSnapshot: true,
```

Comment:
Could we phrase opts in a way that the defaults are the success path? I'm more interested in tests expressing what fails instead of what works.